src/components/variables.css

```diff
@@ -7,3 +7,7 @@
 :root .theme-dark {
   --search-overlays-semitransparent: rgba(42, 46, 56, 0.66);
 }
+
+:root {
+  --editor-footer-height: 27px;
+}
```

Comment:
Hmm, we don't have many variables. I could see adding this variable to the main editor class so that it is scoped, but not sure if that's common practice